bin/storm.py

```diff
@@ -94,7 +94,7 @@ def init_storm_env():
 CONFFILE = ""
 JAR_JVM_OPTS = shlex.split(os.getenv('STORM_JAR_JVM_OPTS', ''))
 JAVA_HOME = os.getenv('JAVA_HOME', None)
-JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java')
+JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java' if not is_windows() else "java.exe")
```

Comment:
I've put the code into a new function. I'm a little uncertain what you're asking in the rest of the comment. The java binary on Windows is called java.exe, so `<JAVA_HOME>/bin/java vs <JAVA_HOME>\bin\java.exe` is correct. The previous implementation should do that?